internal/core/services/claims.go

```diff
@@ -625,9 +626,34 @@ func (c *claim) createVC(ctx context.Context, claimReq *ports.CreateClaimRequest
 }
 
 func (c *claim) guardCreateClaimRequest(req *ports.CreateClaimRequest) error {
-	if _, err := url.ParseRequestURI(req.Schema); err != nil {
-		return ErrMalformedURL
+	type guardFunc func() error
+
+	guards := []guardFunc{
+		// check if schema's URL is valid
+		func() error {
+			if _, err := url.ParseRequestURI(req.Schema); err != nil {
+				return ErrMalformedURL
+			}
+			return nil
+		},
+		// check if refresh service has supported type
+		func() error {
```

Comment:
If RefreshService is set, expiration should be mandatory in that case, right? I'm missing that validation here

Taken from the docs: "Also, the issuer should set expiration field"

Reply:
Yes, you are right. I will fix.